### ISSUE TITLE
Improve Enum serialization to be descriptive

### ISSUE THREAD

**Issue by clairekuang** [specklesystems/speckle-sharp#862]
Object classes like `BrepLoop` and `HatchLoop` include enums which are currently being serialized as ints. 
This needs to be aligned with speckle py (ints serialized as strings), and poses 2 issues:

- easy to break backwards compatibility if the order of enums are changed in the Objects kit
- not descriptive: for example, BrepLoopType.Outer = "Outer" instead of "1" would be much more useful

Proposal: serialize all enums as descriptive strings, figure out any related deserialization issues

**Comment by teocomi** [comment]
Makes sense, although we'd need to implement it in a non breaking way as too maky things could go wrong otherwise!

**Comment by teocomi** [comment]
Closing for now as stale.
